Doppler.Sap/Mappers/SaleOrderMapper.cs

```diff
@@ -16,6 +16,7 @@ public static SapSaleOrderModel MapDopplerBillingRequestToSapSaleOrder(BillingRe
             {
                 NumAtCard = billingRequest.PurchaseOrder ?? "",
                 U_DPL_RECURRING_SERV = billingRequest.IsPlanUpgrade ? "N" : "Y",
+                U_DPL_FIRST_PURCHASE = billingRequest.IsFirstPurchase ? "Y" : "N",
```

Comment:
This class (```SaleOrderMapper```) can be removed. Could you please check if the class is used?. For me was replaced by ```BillingForArMapper```.

Reply:
Ok, I see it